src/en/guide/gettingStarted/requirements.adoc

```diff
@@ -1,9 +1,12 @@
 Before installing Grails you will need a Java Development Kit (JDK) installed with the minimum version denoted in the table below. Download the appropriate JDK for your operating system, run the installer, and then set up an environment variable called `JAVA_HOME` pointing to the location of this installation.
 
+We recommend Liberica JDK 17 LTS: https://bell-sw.com/pages/downloads/#jdk-17-lts
```

Comment:
Should we mention that we recommend this because Spring recommends it on their website (https://spring.io/quickstart) ?

Reply:
Updated